lib/pf/scan.pm

```diff
@@ -253,18 +267,18 @@ sub run_scan {
     # Instantiate the new scan object
     my $scan = instantiate_scan_engine($type, %scan_attributes);
 
-    # Start the scan
+    # Start the scan (it return the scan_id if it failed)
     my $failed_scan = $scan->startScan();
     
     # Hum ... somethings wrong in the scan ?
     if ( $failed_scan ) {
```

Comment:
Should we log a message that it failed here or it's already handled in another part of the code ?